pandas/core/base.py

```diff
@@ -738,22 +740,22 @@ def data(self):
     @property
     def itemsize(self):
         """ return the size of the dtype of the item of the underlying data """
-        return self._values.itemsize
+        return self._ndarray_values.itemsize
 
     @property
     def nbytes(self):
         """ return the number of bytes in the underlying data """
-        return self._values.nbytes
+        return self._ndarray_values.nbytes
```

Comment:
Should `nbytes` be called on `self._values` instead? (as we require a `nbytes` implementation on the Extension Array, and numpy arrays already have it)

Reply:
I think this caused issues for CI, but re-running tests now with this change.